trunk/admin.widget.class.php

```diff
@@ -257,7 +263,7 @@ public function render_sending_method_field()
     public function render_enable_tracking_field()
     {
         printf(
-            '<label><input type="checkbox" id="enable_tracking" name="sp_settings[enable_tracking]" value="1" %s />Track clicks/opens in SparkPost</label>', $this->options['enable_tracking'] ? 'checked' : ''
+            '<label><input type="checkbox" id="enable_tracking" name="sp_settings[enable_tracking]" value="1" %s />Track clicks/opens in SparkPost</label>', @$this->options['enable_tracking'] ? 'checked' : ''
```

Comment:
The alternative is using `SparkPost::get_option` which will give you default value when absent.